docs/source/usage/deployment.rst

```diff
@@ -0,0 +1,118 @@
+    
+       version: 0.2
+       
+       env:
+         variables:
+           STAGE: prod
+           SERVICE: zoom
+           PROJECT: MozDef-EF
+           API_PATH: zoom
+       
+       phases:
+         install:
+           runtime-versions:
+             python: 3.7
+             nodejs: 10
+           commands:
+             # Install dependencies here
+             - pip3 install --upgrade awscli -q
+             - pip3 install --upgrade pytest -q
+             - pip3 install --upgrade moto -q
+             - pip3 install --upgrade aws-xray-sdk -q
+             - npm install -g --silent --progress=false serverless
+             - npm install --silent --save-dev serverless-pseudo-parameters
+             - npm install --silent --save-dev serverless-prune-plugin
+         pre_build:
+           commands:
+             # Perform pre-build actions here
+             - chmod +x $CODEBUILD_SRC_DIR/config/deploy.sh
+             - $CODEBUILD_SRC_DIR/config/deploy.sh unit-test
+         build:
+           commands:
+             # Invoke the deploy script here
+             - $CODEBUILD_SRC_DIR/config/deploy.sh deploy $STAGE $AWS_REGION
+
+
+  3. The important part here is the filling in of the “env” section on top. These environment variables will be used by the “serverless.yml” file when deployed by the serverless framework.
+     The idea is, for each different source (such as zoom), the service name and API path will be different.
+  4. Save the file.
+  5. Make any other desired changes on the local copy. For the zoom service, there should not be any other changes.
+
+*********************
+Deploy Your Framework
+*********************
+
+This is where we take everything we've done up to this point and start the deployment.
+
+  1. Go back to AWS Console  “Services -> CodeCommit” and create a repository with the name “<project>-<service>”, in this case “mozdef-ef-zoom”. Add a description and a tag with key: Project and value: MozDef-EF-prod.
+  2. Using the connection settings, setup Git access with the git credential helper over HTTPS (ensure you can pull and push to the newly created repo)
+  3. Pull the empty repository to a local directory, then add/move all the cloned and updated framework code to this repository. Add and commit all changes, then push.
+  4. Go to “Services -> CloudFormation” on the AWS Console.
+  5. On top right, click “Create stack (with new resources)”
+  6. Select “template is ready” on the first option. In “specify template” menu, select “upload a template file”
+  7. Browse the filesystem, and select the CloudFormation template you wish to use under the “templates” directory of the cloned and updated framework code. Assuming no syntax errors, click next.
+  8. For the stack name, enter something like: “servicename2mozdef”
+  9. For stack parameters, enter the values decided in "Getting Started" Step 2.
+  10. For service, enter your servicename that you determined earlier in this process.
```

Comment:
Here we can say: `For service, enter the value decided for the service in "Getting Started" Step 2.